module.g.cpp containers unreferenced formal parameter when generating with no statics.

When no statics are used the code generated for winrt_get_activation_factory() inside of module.g.cpp declares a parameter and never uses it.  This results in C4100 at compile time.

module.g.cpp(17) : error C4100: 'name': unreferenced formal parameter

example codegen:
void* WINRT_CALL winrt_get_activation_factory(std::wstring_view const& name)
{
    auto requal = [](std::wstring_view const& left, std::wstring_view const& right) noexcept
    {
        return std::equal(left.rbegin(), left.rend(), right.rbegin(), right.rend());
    };

    return nullptr;
}

--- a/src/tool/cppwinrt/cppwinrt/component_writers.h
+++ b/src/tool/cppwinrt/cppwinrt/component_writers.h
@@ -149,7 +149,7 @@ bool __stdcall %_can_unload_now() noexcept
     return true;
 }
 
-void* __stdcall %_get_activation_factory(std::wstring_view const& name)
+void* __stdcall %_get_activation_factory([[maybe_unused]] std::wstring_view const& name)
 {
     auto requal = [](std::wstring_view const& left, std::wstring_view const& right) noexcept
     {

